aries-core/src/js/components/external/stories/Accordion.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Accordion, AccordionPanel, Box, Grommet } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
```

Comment:
Yikes! --- this import will get cleaned up when this work is completed: https://github.com/hpe-design/design-system/issues/107

Reply:
Yes (thanks goodness!)